go/runtime/host/tdx/cid_pool.go

```diff
@@ -0,0 +1,55 @@
+package tdx
+
+import (
+	"fmt"
+	"sync"
+)
+
+// CidPool is a pool of CIDs that can be assigned to VMs.
```

Comment:
```suggestion
// CidPool is a pool of Content IDs (CIDs) that can be assigned to Virtual Machines (VMs).
```

Reply:
These are actually Context IDs, let me update the comment.

Reply:
Ups. That is why it is good to define abbreviation before you start using it. And for me the best place is in the struct's comment.

Reply:
I don't see any change.

Reply:
What do you mean, it has not yet been pushed?

Reply:
Oh, you pushed without this change. I thought it was already fixed in [143d30b](https://github.com/oasisprotocol/oasis-core/pull/6159/commits/143d30b29e4e0b055d11e06915099d0d97fc0dd6).

Reply:
Pushed now.